django/db/models/query_utils.py

```diff
@@ -90,22 +90,26 @@ def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize
         clause, _ = query._add_q(self, reuse, allow_joins=allow_joins)
         return clause
 
-    def refs_aggregate(self, existing_aggregates):
-        def _refs_aggregate(obj, existing_aggregates):
-            if not isinstance(obj, tree.Node):
-                aggregate, aggregate_lookups = refs_aggregate(obj[0].split(LOOKUP_SEP), existing_aggregates)
-                if not aggregate and hasattr(obj[1], 'refs_aggregate'):
-                    return obj[1].refs_aggregate(existing_aggregates)
+    @classmethod
+    def _refs_aggregate(cls, obj, existing_aggregates):
+        if not isinstance(obj, tree.Node):
+            aggregate, aggregate_lookups = refs_aggregate(obj[0].split(LOOKUP_SEP), existing_aggregates)
+            if not aggregate:
+                obj_refs_aggregate = getattr(obj[1], 'refs_aggregate', None)
```

Comment:
I don't like this. the variable name suggests that this is a boolean value, whether ``obj`` references an aggregate, when it's actually a bound method. Is the extra lookup such a performance problem? The existing code uses the same pattern e.g. https://github.com/charettes/django/blob/2bf8315e09aa0e733a84fa2f8efbe67c473f0785/django/db/models/sql/query.py#L1230